solr/src/main/conf/schema.xml

```diff
@@ -0,0 +1,613 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+<!--
+ Licensed to the Apache Software Foundation (ASF) under one or more

```

Comment:
The field name/quantity selection happens above the `DB` implementation layer (this code), namely in workload code. Why would we couple schema definition here to what the default workload does by default? Someone might create another workload config with a different field number - wouldn't this schema get out of sync?

Also, does Solr really requires a fixed set of user-level field definitions?